app/models/review.rb

```diff
@@ -0,0 +1,12 @@
+class Review < ApplicationRecord
+  belongs_to :product
+
+    validates :rating, presence: true
+    validates :rating, numericality: { only_integer: true }
+    validates :rating, numericality: { greater_than_or_equal_to: 1 }
+    validates :rating, numericality: { less_than_or_equal_to: 5 }
```

Comment:
You can collapse this to one line:
```
validates :rating, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 1, less_than_or_equal_to: 5 }
```

The only way this matters is because otherwise you get multiple entries on flash that talk about the errors on numericality on review. I've attached screenshots of before the suggestion and after the suggestion
<img width="495" alt="screen shot 2017-11-02 at 1 57 13 pm" src="https://user-images.githubusercontent.com/30602862/32349921-efcf3154-bfd5-11e7-8f6f-cf48390ed6bf.png">
<img width="487" alt="screen shot 2017-11-02 at 1 57 50 pm" src="https://user-images.githubusercontent.com/30602862/32349928-f6e7ef30-bfd5-11e7-8bba-1a863a505e81.png">